mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -41,7 +41,7 @@ CLASS net/minecraft/class_2056 net/minecraft/item/ItemStack
 	METHOD method_8336 addEnchantment (Lnet/minecraft/class_64;I)V
 		ARG 1 enchantment
 		ARG 2 level
-	METHOD method_8337 onBlockBroken (Lnet/minecraft/class_99;Lnet/minecraft/class_160;IIILnet/minecraft/class_2674;)V
+	METHOD method_8337 onBlockMinedByPlayer (Lnet/minecraft/class_99;Lnet/minecraft/class_160;IIILnet/minecraft/class_2674;)V
```

Comment:
Not sure on this either, sounds more like the `ItemStack` is being mined rather than some external block, at least to me.

Reply:
That's fair. It's a tough method to name tbh. `onBrokeBlock`? `onItemBrokeBlock`? `onPlayerBrokeBlockWithItem`? idk man

Reply:
`onMinedWith`?

Reply:
That works. Perhaps `onMinedBlockWith` or would that be redundant?

Reply:
I think it's redundant, is there anything mineable that isn't a block?